src/main/java/uk/gov/hmcts/ccd/domain/service/aggregated/AuthorisedGetEventTriggerOperation.java

```diff
@@ -63,7 +69,7 @@ public AuthorisedGetEventTriggerOperation(@Qualifier("default") final GetEventTr
     public CaseEventTrigger executeForCaseType(String caseTypeId, String eventTriggerId, Boolean ignoreWarning) {
         final CaseType caseType = caseDefinitionRepository.getCaseType(caseTypeId);
 
-        Set<String> userRoles = caseAccessService.getUserRoles();
+        Set<String> userRoles = Sets.union(caseAccessService.getUserRoles(), singleton(GlobalCaseRole.CREATOR.getRole()));
```

Comment:
can we move `singleton(GlobalCaseRole.CREATOR.getRole())` into CaseAccessService?
e.g.
`caseAccessService.getCaseCreationCaseRoles()`